README.md

```diff
@@ -1,24 +1,208 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="https://user-images.githubusercontent.com/84629565/202665566-ba1a8ed3-041f-45bc-b21b-efdcc357189b.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] OPTIONAL
Kindly ensure to remove the Microverse template logo. Optionally, you can replace it with your own logo if you have one.